LICENSE.md

```diff
@@ -24,20 +23,22 @@ library is linked to its license below:
 
 ## API Server
 * Data Provider modules - some of this code is released under GPLv3
-* [Aura.DI](https://github.com/auraphp/Aura.Di/blob/develop-2/LICENSE)
+* [Aura.DI](https://github.com/auraphp/Aura.Di/blob/4.x/LICENSE)
 * [Flysystem](https://github.com/thephpleague/flysystem/blob/master/LICENSE)
 * [gisconverter](https://github.com/symm/gisconverter/blob/master/LICENSE.txt)
 * [KO3-Event](https://github.com/dkobia/KO3-Event/blob/master/README.md)
-* [Kohana Framework & Ohanzee Components](http://kohanaframework.org/license)
+* [Kohana Framework & Ohanzee Components](https://github.com/koseven/koseven/blob/master/LICENSE.md)
 * [Kohana-Imagefly](https://github.com/Bodom78/kohana-imagefly/blob/master/LICENSE)
-* [Nexmo](https://github.com/prawnsalad/Nexmo-PHP-lib) **To be removed in next release**
-* [OAuth2-Server](https://github.com/thephpleague/oauth2-server/blob/develop/license.txt)
+* [Nexmo](https://github.com/prawnsalad/Nexmo-PHP-lib/blob/master/LICENSE) **(To be removed in next release)**
+* [OAuth2-Server](https://github.com/thephpleague/oauth2-server/blob/master/LICENSE)
 * [Phinx](https://github.com/robmorgan/phinx/blob/master/LICENSE)
 * [Swift Mailer](https://github.com/swiftmailer/swiftmailer/blob/master/LICENSE)
 * [Twilio](https://github.com/twilio/twilio-php/blob/master/LICENSE)
 
-Map Data
-========
+# Map Data
 
-The default map data used comes from OpenStreetMap - <http://osm.org/copyright>
-The default imagery comes from MapQuest and is subject to the following terms: <http://developer.mapquest.com/web/info/terms-of-use> and <http://info.mapquest.com/terms-of-use/>
+The default map data used comes from [OpenStreetMap](http://osm.org/copyright).
+The default imagery comes from MapQuest and is subject to the following terms: 
+- [General](https://hello.mapquest.com/terms-of-use),
+- [For Developers](https://developer.mapquest.com/legal)
```

Comment:
For me, it's not clear, what "for developers" means in this context.

These terms-of-use start with "If you choose to use any of the MapQuest Services, you, (or if applicable, the business you represent) (“Customer”) are agreeing to abide by the conditions and conditions of this MapQuest Enterprise End User License Agreement".

I would interpret this as: if I setup an ushahidi server using MapQuest, these terms do apply. "Developer" sound for me like someone who is contributing code. Is this correct?

Reply:
Yes actually two different `term of use` were written in the LICENSE previously which i am trying to amend. One for `general` and additionally another for `developers`. So i have included both as written in older version on LICENSE.md file.

Reply:
`for developer` is term of use which developers need to understand and follow, if they are in any way using the service.